training/deepspeech_training/util/audio.py

```diff
@@ -340,6 +346,102 @@ def read_opus(opus_file):
     return audio_format, bytes(audio_data)
 
 
+def read_ogg_opus(ogg_file):
+    error = ctypes.c_int()
+    ogg_file_buffer = ogg_file.getbuffer()
+    ubyte_array = ctypes.c_ubyte * len(ogg_file_buffer)
+    opusfile = pyogg.opus.op_open_memory(
+        ubyte_array.from_buffer(ogg_file_buffer),
+        len(ogg_file_buffer),
+        ctypes.pointer(error)
+    )
+
+    if error.value != 0:
+        raise ValueError(
+            ("Ogg/Opus buffer could not be read."
+             "Error code: {}").format(error.value)
+        )
+
+    channel_count = pyogg.opus.op_channel_count(opusfile, -1)
```

Comment:
so we support multi channel?

Reply:
We've had since the dataset refactor. They get converted to 16kHz mono WAV during the augmentation steps.

Reply:
ok, it was unclear in the code here, but indeed I remember seeing the resampling phase

Reply:
Yeah it's not super obvious, it happens here: https://github.com/reuben/DeepSpeech/blob/fcbd92d0d75beee36473aa44669fa330ca522cdc/training/deepspeech_training/util/augmentations.py#L165

(That gets called even if no augmentations are used)

Reply:
Hmmm I don't see where `change_audio_type` ends up performing resampling or merges channels. Can you help me out?

Reply:
Hmmm I don't see where `change_audio_type()` ends up performing resampling or merges channels.